### ISSUE TITLE
Různé/chybné výsledky ve statistikách po přechodu na nové rozhraní

### ISSUE THREAD

**Issue by luckajirku** [ceskaexpedice/kramerius#754]
Při zpracování statistiky návštěvnosti Krameria za minulý rok jsme narazili na pár problémů. Porovnáním výstupů statistik z různých měsíců jsme zjistili, že za stejné období dostáváme rozdílná čísla. 

Issues: 
1) Kramerius K6 neloguje zobrazení stránek do statistik. 
V půli minulého roku (konec června až začátek července) jsme přecházeli na K6 rozhraní. Od té doby návštěvnost spadla z desítek tisíc jen na nízké jednotky tisíc za celý kvartál. To odpovídá našim vlastním potřebám administrace Krameria, provádění kontrol a dalších kroků, ke kterým potřebujeme K4 klienta. Prosíme, prověřit, jestli K6 loguje zobrazení, jak má.

2) Statistiky Krameria uvádějí rozdílná čísla zobrazení pro stejné modely v různých časech. 
Součet zobrazených stran pro první dva kvartály 2019 je o 613 menší, než byl před necelými dvěma hodinami. Součet hodnot čtyř kvartálů je o tisíc menší, než hodnoty za rok, které Kramerius poskytl před pěti minutami. Zároveň je to o asi 700 méně, než před hodinou a půl. Stejné modely, stejné časové rozpětí, stejné aktivity, ale jiná čísla. Proč? Nevíme, kde je chyba, ale nemůžeme mít jinou návštěvnost za již uzavřené období. Buď modul špatně loguje návštěvnost, nebo špatně zobrazuje zalogované výsledky. 

3) Statistiky Krameria jednou uvedly zjevně nepravdivé statistiky. 
Koncem července 2019, po nasazení K6 klienta, Kramerius poskytl statistiky zobrazení stránek za první pololetí. Ty uváděly lehce přes 200 000 zobrazených stránek, což je i při nejlepší vůli zhruba dvojnásobek toho, čeho by šlo dosáhnout. Proč?

Tabulky zobrazených stránek:

za dobu		získané	z Krameria	typ zobrazených	    počet 
(měsíc, rok)	(datum, čas)		stran		    stran	
===============================
1-6 2019	 6. 1. 2020 8:00	veřejné		 82 866
1-6 2019	 6. 1. 2020 8:00	neveřejné	  6 730
===============================
1-6 2019	 6. 1. 2020 7:40	všechny		 89 596
7-12 2019	 6. 1. 2020 7:40	všechny		 26 478
=================================
1-3 2019	 1. 4. 2019		všechny		 41 478		
1-3 2018	 1. 4. 2019		všechny		 28 384
==================================
1-3 2019	 6. 1. 2020 8:30	všechny		 41 478
1-3 2018	 6. 1. 2020 8:30	všechny		 28 384
==================================
1-12 2019	 6. 1. 2020 8:35	všechny		116 372
====================================
1-3 2019	 6. 1. 2020 8:45	všechny		 41 478
4-6 2019	 6. 1. 2020 8:45	všechny	 	 47 505	
7-9 2019	 6. 1. 2020 8:45	všechny		 18 979
10-12 2019	 6. 1. 2020 8:45	všechny	 	  7 348		
====================================
1-6 2019	30. 7. 2019		veřejné		189 019
1-6 2019	30. 7. 2019		neveřejné	 15 999
====================================

**Comment by pavel-stastny** [comment]
@luckajirku Je stále aktuální ?

**Comment by luckajirku** [comment]
@pavel-stastny ano, i po téměř 4 letech, kdy to tu bez povšimnutí leží:-) a v novější verzi 5.8.4.4 problém trvá*. Budeme doufat, že v K7 už budou statistiky fungovat normálně.
*Pokusy se statistikami provedeny v 12:30 a 13:10. Bylo vybráno časové omezení na minulý rok a všechny akce s monografiemi, výsledek byl vyexportován do CSV. V 12:30 dokument obsahoval 129 195 akcí, v 13:10 obsahoval 145 078 akcí.

**Comment by zabak** [comment]
@luckajirku prosím podívej se na aplikaci statistiky NDK, jestli jsou tam všechny potřebné informace. Myslím si, že bychom ji měli považovat pro zapojené knihovny za autoritativní.

**Comment by luckajirku** [comment]
@zabak počkám si, až budeme mít u nás K7, a porovnám si to.
